demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1192,17 +1141,18 @@ def process_test_playbook_path(file_path: str, print_logs: bool) -> tuple:
     Returns:
         pair -- first element is a playbook second is a script. each may be None
     """
-    if print_logs:
-        print("adding {} to id_set".format(file_path))
     script = None
     playbook = None
-    if checked_type(file_path, (TEST_SCRIPT_REGEX, TEST_PLAYBOOK_YML_REGEX, TEST_PLAYBOOK_REGEX)):
-        yml_data = get_yaml(file_path)
-        if 'commonfields' in yml_data:
-            # script files contain this key
+    try:
+        if print_logs:
+            print(f'adding {file_path} to id_set')
+        if find_type(file_path) == FileType.TEST_SCRIPT:
```

Comment:
What is `TEST_SCRIPT` and how it is different from `SCRIPT`?

Reply:
`FileType.TEST_SCRIPT` is the same as `FileType.SCRIPT` with the additional check of the file being in the `TestPlaybooks` folder.

same goes for `FileType.TEST_PLAYBOOK` and `FileType.PLAYBOOK`.